src/main/java/com/icthh/xm/ms/entity/service/XmEntitySpecService.java

```diff
@@ -402,12 +336,12 @@ public TypeSpec findTypeByKey(String key) {
     @SneakyThrows
     private void processUniqueFields(Map<String, TypeSpec> types) {
         for (TypeSpec typeSpec: types.values()) {
-            if (isBlank(typeSpec.getDataSpec())) {
+            if (StringUtils.isBlank(typeSpec.getDataSpec())) {
                 continue;
             }
 
             JsonNode node = JsonLoader.fromString(typeSpec.getDataSpec());
-            Set<UniqueFieldSpec> uniqueFields = new HashSet<>();
+            Set<UniqueFieldSpec> uniqueFields = Sets.newHashSet();
```

Comment:
We in java 6?
why we try to do what recommended for java 6?
from java 7 we didn't have reason to do it!

Reply:
It is personal preference. Actually you are doing it Optional.of() and other static factories. Josh Block recommendation is still fine for Java. Constructor versus Factory is not 6,7,8 discussion.